src/middleware.ts

```diff
@@ -43,14 +43,6 @@ export function middleware(request: NextRequest) {
     },
   });
 
-  if (!existingExperimentationId) {
-    response.cookies.set({
```

Comment:
Why remove this?

Reply:
I think the point of setting the cookie is for users that are not authenticated, so there's no way to keep track of which experiments or roll-outs they are enrolled in otherwise.